maquetacion/index.html

```diff
@@ -11,6 +11,16 @@
 <body>
   <div class="content">
     <h1 class="title">Front-End Tools</h1>
+    <div class="content-cards">
+      <div class="card">
+        <img class="card-image" src="assets/react_icon.svg" alt="React icon">
+        <h3 class="card-title">ReactJS</h3>
+        <h3 class="card-title fw-bold">Release: 2013</h3>
```

Comment:
I think this string has different font size than the main title ("ReactJS"). Maybe you can use a span here. What do you think?